apps/modernization-ui/src/apps/search/laboratory-report/result/list/LaboratoryReportSearchResultListItem.tsx

```diff
@@ -1,9 +1,187 @@
-import { LabReport } from 'generated/graphql/schema';
+import {
+    Jurisdiction,
+    LabReport,
+    LabReportOrganizationParticipation,
+    useFindAllJurisdictionsLazyQuery
+} from 'generated/graphql/schema';
+import styles from './LaboratoryReportSearchResultListItem.module.scss';
+import { Link } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 
 type Props = {
     result: LabReport;
 };
 
-const LaboratoryReportSearchResultListItem = ({ result }: Props) => <>{result.localId}</>;
+const LaboratoryReportSearchResultListItem = ({ result }: Props) => {
+    const [jurisdictions, setJurisdictions] = useState<any[] | null>(null);
+
+    const noData = 'No data';
+    const patient = result.personParticipations.find((p) => p.typeCd === 'PATSBJ');
+    const firstName = `${patient?.firstName}` ?? null;
```

Comment:
Nit: is `const firstName = patient?.firstName ?? null;` better? The string interpolation seems unnecessary.